src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/Example4.java

```diff
@@ -9,16 +9,19 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.coding.finallocalvariable;
+
 // xdoc section -- start
-public class MyClass {
-  static int foo(int x, int y) { //violations, parameters should be final
+class Example4
+{
+  static int foo(int x, int y) { // 2 violations
```

Comment:
please review images of issue you fixing.
Them try to show that all Examples needs to share  the "same" code, so we need to kind of merge all example in one code and used for all exmples.

Reply:
> 2 violations

please use this pattern to show vilations
https://github.com/checkstyle/checkstyle/blob/bf31b649f02bc0b68ba9a8fdc79cb0b090db0968/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/Example1.java#L25-L28

Reply:
done

Reply:
Done only for `2 violations` .

Please make a diff of Example1 and Example2. And  make sure that diff is minimal as shown in issue images.

Reply:
Do you mean the same pattern in both Example1 and Example2 ? 
Because as far as I can see in the issue images it is like the same thing but with the addition of `,`
example : `violation , ...` .. Like this ?

Reply:
Exactly. Examples should be the same by structure, only comments are different

Reply:
done

Reply:
My bad now it's done
I have restructured all the examples to be the same